File: app/src/main/java/com/parassidhu/cdlumaths/models/GetSetGo.java
package com.parassidhu.cdlumaths.models;


public class GetSetGo {
    String key;
    String value;
    String rollno;

    public GetSetGo(String key, String value) {
        this.key = key;
        this.value = value;
    }

    public GetSetGo(String key, String value, String rollno) {
        this.key = key;
        this.value = value;
        this.rollno = rollno;
    }

    public String getRollno() { return rollno; }

    public String getKey() {
        return this.key;
    }

    public void setKey(String key) {
        this.key = key;
    }

    public String getValue() {
        return this.value;
    }

    public void setValue(String value) {
        this.value = value;
    }
}

File: app/src/main/java/com/parassidhu/cdlumaths/models/GetSetGo.kt
package com.parassidhu.cdlumaths.models


class GetSetGo(var key: String, var value: String) {
    lateinit var rollNo: String
        internal set

    constructor(key: String, value: String, rollNo: String) : this(key, value) {
        this.rollNo = rollNo
    }
}
